models/Repository.ts

```diff
@@ -1,4 +1,173 @@
-export interface IRepository {
+export interface GitHubResponse {
```

Comment:
Do we really need to map the entire github response? We don't use most of the fields here